.pre-commit-hooks.yaml

```
@@ -15,7 +15,7 @@
   language: script
   # Supported extensions by Terraform specified in
   # https://github.com/hashicorp/terraform/blob/0c63fb2b097edcd5cb1a91322765a414206fbea2/internal/command/fmt.go#L30-L35
-  files: \.(tf|tofu|tfvars|tftest\.hcl|tfmock\.hcl)$
+  files: \.(tf|tofu|tfvars|(tftest|tofutest|tfmock)\.hcl)$
```

```suggestion
  files: \.(tf|tofu|tfvars|(tftest|tofutest|tfmock|tfquery)\.hcl)$
```
And as we here, let's also add support for `.tfquery.hcl`, as it will be introduced in TF 1.14
https://github.com/hashicorp/terraform/blob/eae5ac13dd47eafd2d338b21db7311bbfa861b68/internal/command/fmt.go#L56

https://github.com/hashicorp/terraform/releases/tag/v1.14.0-beta2

---

@tvanriel can you please merge lastest changes from master and commit suggestion above? Thank you